knowledge_graph/make_graph.py

```diff
@@ -345,30 +190,26 @@ def makeGraph(onto_path, edge_path, output_folder_path):
     # print(list(default_world.inconsistent_classes()))
 
     # Read in the triples data
-    df = pd.read_csv(edge_path)
+    ## DMARX - csv via make_network.outputEdges()
```

Comment:
The CSV file was more for us to check that our graph traversal code was working correctly. I think it's ok to remove the CSV file step completely from the pipeline. And to move things to the network class as needed like you suggest.

Reply:
I don't mind outputting a CSV file at this step. The issue I had was that if there was that passing the edges from one phase of the process to another using this file felt like it was creating an opportunity wherein if the first part of the process failed silently, the second part would load the old CSV file and it would be difficult to debug what happened.

Reply:
Let's take out the part where it reads in the edges from the CSV file. Have it read in edges from the python object directly from the previous step in the pipeline. It can still save a csv for record keeping. Just don't want it to read directly from it, right?

Reply:
yup, exactly what I had in mind!

Reply:
Please do this and the other changes we talked about that are still needed and then I can approve your PR